packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/edit-display-name/controller.js

```diff
@@ -3,6 +3,8 @@ export default class EditNameCtrl {
   constructor($translate, NutanixNode) {
     this.$translate = $translate;
     this.NutanixNode = NutanixNode;
+    // eslint-disable-next-line no-control-regex
+    this.namePattern = new RegExp('^[\x00-\x7F]{1,40}$');
```

Comment:
```suggestion
```
maybe we can move to constant file for to reuse for the second place where is used in packages/manager/modules/nutanix/src/dashboard/general-info/edit-display-name/controller.js